### ISSUE TITLE
Косячное отображение объемов Bid/Ask 

### ISSUE THREAD

**Issue by BearTrading** [SBProX/SBProX-Dev#1661]
![photo_2020-11-06_08-23-34](https://user-images.githubusercontent.com/65649287/98339460-ca4ab980-201c-11eb-9a07-0add748809d1.jpg)
![photo_2020-11-06_08-31-10 (2)](https://user-images.githubusercontent.com/65649287/98339462-cae35000-201c-11eb-9e82-a8b404a52836.jpg)
![photo_2020-11-06_08-31-10](https://user-images.githubusercontent.com/65649287/98339463-cb7be680-201c-11eb-9c5c-663323cc2480.jpg)

Имеется ли возможность реализации подобного вида Bid/Ask профиля, как на первом скрине? То, что реализовано в настоящий момент крайне не удобно и топорно + при уменьшении графика исчезают цифровые данные, так же скрин прикладываю.

**Comment by Vitalyi95** [comment]
Поддерживаю, было бы намного удобней!

**Comment by SBProX** [comment]
![image](https://user-images.githubusercontent.com/16720120/100126468-a396da80-2e86-11eb-8c03-bab2db153771.png)

Вышло обновление

**Comment by SBProX** [comment]
![Uploading image.png…]()

**Comment by SBProX** [comment]
![image](https://user-images.githubusercontent.com/16720120/100126711-fcff0980-2e86-11eb-94a9-74226871b043.png)

**Comment by BearTrading** [comment]
Да, видел ребят) Проделана хорошая работа) Уже отписался в Тех поддержку)
